Calculator/Calculator/Model/Extensions/String+Extension.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  String+Extension.swift
+//  Calculator
+//
+//  Created by Minseong Kang on 2023/06/06.
+//
+
+import Foundation
+
+extension String {
+	func split(with target: Character) -> [String] {
+		let splitByTarget = self.split(separator: target).map { String($0) }
+
+		return splitByTarget
```

Comment:
이부분 역시 굳이 상수로 선언한 후에 반환할 필요는 없어보입니다!
```suggestion
		return self.split(separator: target).map { String($0) }
```